app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,412 @@
+  top: 3px;
+  vertical-align: middle;
+  -webkit-box-shadow: inset 0 1px 0 0 #cdc6c6, inset 0 -1px 0 0 #d9d4d4;
+  left: 0;
+  width: 200px;
+  outline: none;
+  height: 5px;
+  margin: 0;
+  padding: 0;
+}
+div.range-slider input[type="range"]::-webkit-slider-thumb {
+  pointer-events: all;
+  position: relative;
+  z-index: 1;
+  outline: 0;
+  -webkit-appearance: none;
+  box-shadow: 0px 0px 5px black;
+  width: 14px;
+  height: 14px;
+  border: 2px black;
+  border-radius: 14px;
+  background-image: -webkit-radial-gradient(black 20%, white 20%, white 98%, black 98%);
+  /* android <= 2.2 */
+  background-image: -webkit-radial-gradient(black 20%, white 20%, white 98%, black 98%);
+  /* older mobile safari and android > 2.2 */
+  background-image: radial-gradient(black 20%, white 20%, white 98%, black 98%);
+  /* W3C */
+}
+div.range-slider input[type="range"]::-moz-range-thumb {
+  pointer-events: all;
+  position: relative;
+  z-index: 10;
+  -moz-appearance: none;
+  width: 20px;
+  height: 20px;
+  border: 2px black;
+  box-shadow: 0px 0px 20px black;
+  border-radius: 14px;
+  background-image: radial-gradient(black 20%, white 20%, white 98%, black 98%);
+  /* W3C */
+}
+div.range-slider input[type="range"]::-ms-thumb {
+  pointer-events: all;
+  position: relative;
+  z-index: 10;
+  -ms-appearance: none;
+  width: 20px;
+  height: 20px;
+  border-radius: 14px;
+  border: 2px black;
+  background-image: radial-gradient(black 20%, white 20%, white 98%, black 98%);
+  /* W3C */
+}
+div.range-slider input[type=range]::-moz-range-track {
+  position: relative;
+  z-index: -1;
+  background-color: black;
+  border: 0;
+}
+div.range-slider input[type=range]:last-of-type::-moz-range-track {
+  -moz-appearance: none;
+  background: none transparent;
+  border: 0;
+}
+div.range-slider input[type=range]::-moz-focus-outer {
+  border: 0;
+}
+/* footer */
+
+footer {
+  margin-top: 5px;
+  padding-top: 5px;
+  border-top: 1px solid #eaeaea;
+  color: #777;
+  a {
+    color: #555;
+    &:hover {
+      color: #222;
+    }
+  }
+  small {
+    float: left;
+  }
+  ul {
+    float: right;
+    list-style: none;
+    li {
+      float: left;
+      margin-left: 15px;
+    }
+  }
+}
+#price-tag {
+  letter-spacing: 2px;
+  font-size: 9px;
+  font-weight: 900;
+  font-family: Verdana !important;
+  color:orange;
+  padding-top: 25%
+}
+.card {
+  text-align: center;
+  transition: transform .3s;
+  border-width: 0px;
+  border-radius: 10px;
+  width: 100%;
+  height: 100%;
+  max-width: 250px;
+  margin: 0 auto;
+}
+.card:hover {
+  transform: scale(1.1);
+  box-shadow: 0px 0px 10px black;
+  z-index: 7;
+}
+.card-body .card-text {
+  font-size: 12px;
+  color:gray;
+}
+.card-img-top{
+  border-radius:10px 10px 0 0;
+  overflow: hidden;
+  width:100%;
+  height:50%;
+}
+#product-image {
+  max-width: 100%;
+}
+.card-body .price{
+  font-size:16px;
+  color:orange;
+}
+.card-text {
+  margin-bottom: 5px;
+}
+div.col-md-3 {
+  padding: 5px;
```

Comment:
NEVER overwrite bootstrap classes like this. Instead you could just use `p-2` or whatever class from bootstrap 4.

Reply:
solved. ✓